metric/metric_test.go

```diff
@@ -484,21 +486,26 @@ func TestErrorOnLineTooLong(t *testing.T) {
 			return
 		}
 
-		if !strings.HasPrefix(err.Error(), "Serialized line exceeds limit of 2000 characters accepted by the ingest API:") {
-			t.Errorf("Expected error message to start with 'Serialized line exceeds limit of 2000 characters accepted by the ingest API:' but it did not: %s", err.Error())
+		if !strings.HasPrefix(err.Error(), "Serialized line exceeds limit of 50000 characters accepted by the ingest API:") {
+			t.Errorf("Expected error message to start with 'Serialized line exceeds limit of 50000 characters accepted by the ingest API:' but it did not: %s", err.Error())
 			return
 		}
 
-		// make sure the first key-value pair is present
-		if !strings.Contains(err.Error(), "dim0=val0") {
-			t.Error("Expected the error to contain 'dim0=val0' but it did not.")
-			return
+		dimensionRegex, compileErr := regexp.Compile("dim\\d+=val\\d+,")
+		if compileErr != nil {
+			t.Error("failed to compile test regex")
+		}
+
+		if !dimensionRegex.MatchString(err.Error()) {
+			t.Error("Expected the error to contain at least one dimX=valX dimension, where X is any number, but it did not.")
+		}
+
+		if !strings.HasSuffix(err.Error(), "... (truncated)") {
+			t.Errorf("Expected the error to end with '... (truncated)' but it did not.")
```

Comment:
I think this could just be `t.Error()` since it does not format anything, right? :slightly_smiling_face: 
```suggestion
			t.Error("Expected the error to end with '... (truncated)' but it did not.")
```